tests/mock_wp_functions.php

```diff
@@ -151,3 +152,13 @@ function wp_unslash($value)
 {
     return $value;
 }
+
+function wp_enqueue_script($handle, $src = '', $deps = array(), $ver = false, $in_footer = false)
+{
+    MockWpEnqueue::add($handle, $src , $deps , $ver, $in_footer );
```

Comment:
formatting, check ff je phpstorm instellingen

Reply:
Formatting aangepast, het stond nog uit in Phpstorm